src/components/Experiment/QuickInput/QuickMultiInput.js

```diff
@@ -47,22 +47,42 @@ export default function QuickMultiInput(props) {
         </div>
         {tabs.map((tab, tabIndex) => (
           <div key={tabIndex}>
-            {task.inputs.map((input, inputIndex) => (
-              <QuickMultiInputTabContent
-                key={inputIndex}
-                tab={tab}
-                tabIndex={tabIndex}
-                getElement={getElement}
-                {...props}
-                removeInput={removeInput}
-                addInput={addInput}
-                selectInput={selectInput}
-                tabIsSelected={tabIsSelected}
-                selectedInputs={selectedInputs}
-                input={input}
-                inputIndex={inputIndex}  
-              />
-            ))}
+            {task.inputs.map((input, inputIndex) => {
+              const noURLInput = input?.inputUrl === false;
+              const noUploadInput = input?.inputUpload === false;
+              if ((tab.id === "url-input" && noURLInput) || (tab.id === "upload-input" && noUploadInput)) {
+                return (<QuickMultiInputTabContent
+                  key={inputIndex}
+                  tab={input.defaultTab}
```

Comment:
Rather than duplicating the `return <QuickMultiInputTabContent>`, can't you use the if statement above to set a new variable like:
```
let tabObj = tab;
if ((tab.id === "url-input" && noURLInput) || (tab.id === "upload-input" && noUploadInput)) {
  tabObj = input.defaultTab
}
```
(This is just an example, you can def make it cleaner)